docs/designs/flatten_odps_key_value_table.md

```diff
@@ -0,0 +1,107 @@
+# Transform the Column with Key-value Pairs String to Wide Table
+This document describes the design to transform the column with key-value pairs string to a wide ODPS table where
```

Comment:
the design to normalize (or transform) the table schema to be wide, aka one feature per column